packages/api/tests/api/admin/me/get.test.ts

```diff
@@ -0,0 +1,18 @@
+import { get } from '../../../../src/api/admin/me/get';
+
+const mockRequest: any = {
+  user: { id: '1' },
+};
+
+const mockResponse: any = {
+  send: jest.fn(),
+};
```

Comment:
Both of these should be using the helper methods instead of making objects from scratch and both should be within a test instead of at the root of the file